pkg/version/version.go

```diff
@@ -30,6 +30,9 @@ const VersionPrefix = "v"
 // version is a private field and should be set when compiling with --ldflags="-X k8s.io/minikube/pkg/version.version=vX.Y.Z"
 var version = "v0.0.0-unset"
 
+// version is a private field and should be set when compiling with --ldflags="-X k8s.io/minikube/pkg/version.gitCommitId=<commit-id>"
+var gitCommitId = ""
```

Comment:
this should be gitCommitID: https://github.com/golang/go/wiki/CodeReviewComments#initialisms